qa-niaid.planx-pla.net/portal/gitops.json

```diff
@@ -52,231 +52,29 @@
-      "meta": {
-        "condition": "COVID-19",
-        "study_design": "Interventional (Clinical Trial), Randomized Allocation",
-        "sponsor": "National Institute of Allergy and Infectious Diseases (NIAID)",
-        "study_dates": "February 21, 2020 - May 21, 2020",
-        "data_available": "Patient-level data",
-        "websites": [{
-          "name": "Visit ClinicalTrials.gov",
-          "type": "link",
-          "link": "https://clinicaltrials.gov/ct2/show/NCT04280705"
-        }],
-        "study_publications": [{
-          "name": "Remdesivir for the Treatment of Covid-19 - Preliminary Report",
-          "type": "link",
-          "link": "https://pubmed.ncbi.nlm.nih.gov/32445440/"
-        }]
-      },
-      "document": [{
-        "name": "Patient-Level Data Elements",
-        "format": "PDF",
-        "type": "file",
-        "link": "/",
-        "size": 763843
-      }, {
-        "name": "Protocol",
-        "format": "PDF",
-        "type": "file",
-        "link": "/",
-        "size": 7638643
-      }],
-      "hasAccess": false
-    }]
-  },
-  "dataExplorerConfig": {
-    "charts": {
-      "project_id": {
-        "chartType": "count",
-        "title": "Projects"
-      },
-      "submitter_id": {
-        "chartType": "count",
-        "title": "Subjects"
-      },
-      "gender": {
-        "chartType": "pie",
-        "title": "Gender"
-      },
-      "race": {
-        "chartType": "bar",
-        "title": "Race"
-      },
-      "species": {
-        "chartType": "bar",
-        "title": "Species"
-      }
-    },
-    "filters": {
-      "tabs": [{
-          "title": "Subject",
-          "fields": [
-            "project_id",
-            "gender",
-            "race",
-            "ethnicity",
-            "vital_status",
-            "frstdthd",
-            "species",
-            "data_type",
-            "data_format"
-          ]
-        },
-        {
-          "title": "Diagnosis",
-          "fields": [
-            "arthxbase",
-            "bshbvstat",
-            "bshcvstat",
-            "cd4nadir",
-            "status",
-            "virus_type",
-            "virus_subtype",
-            "hiv_status"
-          ]
-        },
-        {
-          "title": "Drug Resistance",
-          "fields": [
-            "amikacin_res_phenotype",
-            "capreomycin_res_phenotype",
-            "isoniazid_res_phenotype",
-            "kanamycin_res_phenotype",
-            "ofloxacin_res_phenotype",
-            "pyrazinamide_res_phenotype",
-            "rifampicin_res_phenotype",
-            "rifampin_res_phenotype",
-            "streptomycin_res_phenotype"
-          ]
-        }
-      ]
-    },
-    "table": {
-      "enabled": false
+    "defaultOpenStudyName": "",
+    "dataType": "cmc",
+    "title": "Studies",
+    "titleField": "title",
+    "rowAccessor": "project_id",
```

Comment:
can we use submitter_id instead

Reply:
doesn't have to be right now

Reply:
yeah just need to get submitter_id into all 3 es indices

can do that in another pr?

Reply:
yeah!